tests/unittests/OperatorTest.cpp

```diff
@@ -3079,6 +3079,302 @@ TEST_P(OperatorTest, TransposeIntoReshapeOptim) {
   }
 }
 
+/// Helper to check the code generation for flip nodes.
+template <typename elemType>
+static void testFlip(glow::PlaceholderBindings &bindings, glow::Module &mod,
+                     glow::Function *F, glow::ExecutionEngine &EE,
+                     std::vector<elemType> inputData,
+                     std::vector<elemType> expectedData,
+                     llvm::ArrayRef<dim_t> dims, dim_t axis,
+                     ElemKind elemKind = ElemKind::FloatTy) {
+
+  // Create network.
+  Placeholder *input;
+  if (isQuantizedElemKind(elemKind)) {
```

Comment:
nit: We have a helper `createPlaceholderConditionallyQuantized()` which I believe does this exact thing.

Reply:
Done.